SmartTutor/Controllers/DTOs/Challenge/ChallengeFulfillmentStrategyDTO.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace SmartTutor.Controllers.DTOs.Challenge
+{
+    public class ChallengeFulfillmentStrategyDTO
```

Comment:
DTOs are Data transfer objects - they are used to move data from the backend to a front-end application such as a the Web or Plugin. The challenge fulfillment logic and all its composing elements are backend logic that should never go to a front-end as it defeats the purpose of the challenge if you know the validation rules.